### ISSUE TITLE
Improve icon

### ISSUE THREAD

**Issue by mipstian** [mipstian/catch#71]
Make it more at home in a post-Yosemite world.

**Comment by studiosciences** [comment]
May I be of assistance?

**Comment by mipstian** [comment]
Sure! I was thinking:
- keeping the same basic shapes and colors
- making the rss symbol a bit thinner and with flat edges (more like the menubar icon)
- removing the gloss
- simplifying the tv a bit, similar to the way the Terminal, Mission Control and Activity Monitor icons look
What do you think?
Also, what sw do you use? It'd be nice to have the source file in the repo for the future. I use Affinity Designer for this kind of stuff.

**Comment by studiosciences** [comment]
I usually use Sketch these days. I'm on vacation now. I'll check in next week. I'll have some time.

PS Sorry, I think I clicked close instead of comment.

**Comment by mipstian** [comment]
Great, thanks!

**Comment by studiosciences** [comment]
<img width="525" alt="screen shot 2015-09-22 at 09 21 46 am" src="https://cloud.githubusercontent.com/assets/1742764/10024332/962ac788-610b-11e5-8a3a-f4615cdec3b1.png">

How flat do you want it?

**Comment by studiosciences** [comment]
Small base vs no base:
<img width="1084" alt="screen shot 2015-09-22 at 09 27 45 am" src="https://cloud.githubusercontent.com/assets/1742764/10024540/82a9ce4c-610c-11e5-9dd4-9ad0fd5cd2b0.png">

**Comment by mipstian** [comment]
Nice! Here's a rebound. I added a gradient so it pops more (and also looks like the tv is on) and made a straight version of the rss logo. It came out a little fatter than I expected :/ but I guess it gets the idea across.
<img width="525" alt="962ac788-610b-11e5-8a3a-f4615cdec3b1" src="https://cloud.githubusercontent.com/assets/421341/10028276/c7b1e588-616b-11e5-90ec-b94b3a30cb0b.png">

**Comment by mipstian** [comment]
About the borders: I think we should figure out how to make them stand out more from the screen - they get lost at small sizes (dock size, launchpad size).

**Comment by studiosciences** [comment]
I like your changes. Will update later today hopefully.

I have a version with thicker borders I'll work on a bit more.

**Comment by mipstian** [comment]
Great to hear! Thanks again!

**Comment by mipstian** [comment]
I just realized why the logo looks weird. Or at least different from the one in the menubar. The two "waves" should be half the size of the dot and of the gaps. So if we look at the bottom line of the logo, it should be:
2x white (dot) - 2x gap - 1x white (mid wave) - 2x gap - 1x white (out wave)
I made everything the same size in my rebound, so of course the waves look fat :)

**Comment by studiosciences** [comment]
<img width="528" alt="screen shot 2015-09-24 at 15 25 29 pm" src="https://cloud.githubusercontent.com/assets/1742764/10088604/cd7c8386-62d0-11e5-8641-6f9afc52e73d.png">

2x for the spacing was too much. Here is 1.618, the golden ratio.
Thicker borders may be too much for the 512px version. I can make a variation for that one.

**Comment by mipstian** [comment]
Looking good! When you get around to fixing the gradient, can you send over a version with a transparent background (or an .icns)?

**Comment by studiosciences** [comment]
The wave is a line with a stroke. It's possible it got a little out of whack at some point. I'll double check the source files again. In the mean time, here's a big image with all the sizes.

I'll put this in the asset catalog and open a pull requests. It's actually ready to go, except for the line tweak you mention.
![demo](https://cloud.githubusercontent.com/assets/1742764/10178443/9786b250-66b4-11e5-9356-4737aa8c629f.png)

**Comment by studiosciences** [comment]
Blech, so I have to remake that logo for the nth time.

**Comment by mipstian** [comment]
I just did it as concentric circles, and then masked it to a square. It's the easiest way I can think of! Can't wait for the PR!

**Comment by studiosciences** [comment]
Yeah, I switched to that method, but my calculations were still off. Plus, I didn't realize Sketch hadn't scaled the stroke. I'm going to post two different line weights before the PR.

**Comment by studiosciences** [comment]
OK last time before I do the Pull Request. Updating those files will only take a few minutes. What are you thoughts on line thickness:

<img width="675" alt="screen shot 2015-09-30 at 07 52 14 am" src="https://cloud.githubusercontent.com/assets/1742764/10196589/87eccfd4-6748-11e5-8525-e9bbeeec3c0d.png">

I made a spreadsheet to do the algebra.

**Comment by mipstian** [comment]
1.5 all the way! :)

I actually like both aesthetically, but I think 1.5 is more immediately recognizable as the RSS symbol.

**Comment by studiosciences** [comment]
Agreed. The 2.0 lines also have a weird resonance with the outer border.

**Comment by mipstian** [comment]
We just need a drop shadow and it's done. Yosemite-style icons have a pretty light drop shadow, but if it's not there at all it really stands out.

**Comment by studiosciences** [comment]
I'll look into the drop shadow over lunch.

**Comment by mipstian** [comment]
![1024](https://cloud.githubusercontent.com/assets/421341/10587106/eab8161c-769f-11e5-90e8-0aa7ea298423.png)

Is this any good? The base is a bit funky. I'd have to work on it more to get it to look nice at all sizes. I think the stronger gradient works really well though - I'd like to see how your version would look like with my gradient.

**Comment by studiosciences** [comment]
I like it a lot, but as you say the stand looks a bit weird. The borders on
the top/sides of the base should be the cool gray perhaps since they would
be lit from above.

You could also thine the border of the base toward the back to make it
appear to recede.

-marc

**Comment by mipstian** [comment]
Ok I'll work on it some more then, thanks for the tips!

**Comment by mipstian** [comment]
Here's what I got so far:
<img width="512" alt="512 2x" src="https://cloud.githubusercontent.com/assets/421341/11056907/e0e85a0a-8786-11e5-8acd-2d96cfcfa7f1.png">
Doesn't look amazing when huge, but I think it looks really good in a normal context (pops, but blends in):
<img width="761" alt="screen shot 2015-11-10 at 8 40 15 am" src="https://cloud.githubusercontent.com/assets/421341/11056922/02fc9a8e-8787-11e5-9f4c-bfdab255861e.png">

I've also made a new icon for the Feed tab in Preferences, because the old one was pre-Yosemite style. I'm not too happy with how it looks, but considering it's not a frequently looked at part of the UI, I'm okay with it
<img width="211" alt="screen shot 2015-11-10 at 8 40 36 am" src="https://cloud.githubusercontent.com/assets/421341/11056947/36c3225c-8787-11e5-996c-5bc7f0610717.png">

All of this is in a new_icon branch if you wanna check it out. Any suggestions?
